src/igvfd/types/file_set.py

```diff
@@ -118,6 +118,88 @@ class CuratedSet(FileSet):
     schema = load_schema('igvfd:schemas/curated_set.json')
     embedded_with_frame = FileSet.embedded_with_frame
 
+    @calculated_property(
+        schema={
+            'title': 'Assembly',
+            'description': 'The genome assembly to which the referencing files in the file set are utilizing (e.g., GRCh38).',
+            'type': 'array',
+            'uniqueItems': True,
+            'items': {
+                'type': 'string'
+            },
+            'notSubmittable': True,
+        }
+    )
+    def assembly(self, request, files):
+        if files is not None:
+            assembly_values = set()
+            for current_file_path in files:
+                file_object = request.embed(current_file_path, '@@object?skip_calculated=true')
+                if file_object.get('assembly'):
+                    assembly_values.add(file_object.get('assembly'))
+        if assembly_values:
+            return list(assembly_values)
+
+    @calculated_property(
+        schema={
+            'title': 'Transcriptome Annotation',
+            'description': 'The annotation and version of the reference resource.',
+            'type': 'array',
+            'uniqueItems': True,
+            'items': {
+                'type': 'string'
+            },
+            'notSubmittable': True,
+        }
+    )
+    def transcriptome_annotation(self, request, files):
+        if files is not None:
+            annotation_values = set()
+            for current_file_path in files:
+                file_object = request.embed(current_file_path, '@@object?skip_calculated=true')
+                if file_object.get('transcriptome_annotation'):
+                    annotation_values.add(file_object.get('transcriptome_annotation'))
+        if annotation_values:
+            return list(annotation_values)
+
+    @calculated_property(
```

Comment:
@calculated_property(assembly=None, transcriptome_annotations=None,  schema={...

Reply:
This results in the following error:

`TypeError: add_calculated_property() got an unexpected keyword argument 'assembly'`

The error is reported 875 times...

Reply:
specifying define=True in the arguments for a calculated property allows passing it in to another calculated property. I have done this in my commit.